crates/wat/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wat"
-version = "1.0.35"
+version = "1.1.0"
```

Comment:
Similar to `wasmprinter` the API of this crate didn't change, so it's safe to just bump to 1.0.36